patientDataViewControl.md

```diff
@@ -0,0 +1,76 @@
+# Patient Data View Control
+
+If there is any specific resources, that the patient needs to exclude from their patient $everything operation, a data view control Consent resource can be made for same. For any resource that is referenced in the Consent, will be excluded from the result.
+
+Data is excluded for patient scoped request only and person inside jwt is matched with the proxy patient in `patient.reference` field.
```

Comment:
If the proxy patient docs are merged before this it might be good to link to those here so a user can understand how these two are related.